### ISSUE TITLE
midvatten settings dialog does not show up when db is missing

### ISSUE THREAD

**Issue by jkall** [jkall/qgis-midvatten-plugin#250]
Found on **postgis-branch**. Not tested on other branches.

Midvatten settings dialogue is not shown if the database is unavailable (e.g. network share not mounted). Hence the user can not select another, available, database without using the rather drastic "reset settings" method. 

It would be preferrable if the settings dialogue could be shown but without any selected database (and no selected tables and columns). I am quite sure this was the previous behaviour. Also, the other settings (custom plot, fieldlogger etc) should only be reset if user selects "Midvatten-Utilities-Reset settings" and not just because database path is temporarily unavailable or has changed. 

```
2017-11-24T09:36:24	1	Traceback (most recent call last):
			  File "/home/josef/.qgis2/python/plugins/midvatten/midvatten.py", line 784, in setup
			    self.midvsettingsdialog = midvsettingsdialog.midvsettingsdialogdock(self.iface.mainWindow(),self.iface, self.ms)#self.iface as arg?
			  File "/home/josef/.qgis2/python/plugins/midvatten/midvsettingsdialog.py", line 43, in __init__
			    self.initUI()
			  File "/home/josef/.qgis2/python/plugins/midvatten/midvsettingsdialog.py", line 50, in initUI
			    self.LoadAndSelectLastSettings()
			  File "/home/josef/.qgis2/python/plugins/midvatten/tools/db_utils.py", line 271, in func_wrapper
			    if check_connection_ok():
			  File "/home/josef/.qgis2/python/plugins/midvatten/tools/db_utils.py", line 260, in check_connection_ok
			    dbconnection = DbConnectionManager()
			  File "/home/josef/.qgis2/python/plugins/midvatten/tools/db_utils.py", line 98, in __init__
			    self.conn = sqlite.connect(self.dbpath, detect_types=sqlite.PARSE_DECLTYPES | sqlite.PARSE_COLNAMES)
			OperationalError: unable to open database file
```

**Comment by HenrikSpa** [comment]
Probably fixed in 1.5.5b. (couldn't reproduce it.)

**Comment by jkall** [comment]
Agreed, probably fixed and hence closing this.
